lib/ret/twitter_client.ex

```diff
@@ -3,6 +3,7 @@ defmodule Ret.TwitterClient do
 
   @twitter_api_base "https://api.twitter.com"
   @twitter_upload_api_base "https://upload.twitter.com"
+  @max_video_upload_size 64 * 1024 * 1024
```

Comment:
Is the lambda substrate fast enough to transcode 64MB within the 30 second cap?

Reply:
i tested with 60s videos, i don't think they made it up to 64MB, but it seems highly likely. it's pretty fast (except the cold start)